integration-tests/api/__tests__/returns/index.js

```diff
@@ -189,6 +189,56 @@ describe("/admin/orders", () => {
     )
   })
 
+  test("creates a store return w. new tax system + shipping", async () => {
+    await adminSeeder(dbConnection)
+    const order = await createReturnableOrder(dbConnection, { oldTaxes: false })
+    const returnOption = await simpleShippingOptionFactory(dbConnection, {
```

Comment:
question: shouldn't this be set up with a tax-inclusive price since that is the scenario we are having issues with?

Reply:
@srindom that is true - however the functionality is relying on `TaxInclusivePricingFeatureFlag` to determine wether the price is tax inclusive and this feature flag is not enabled in integration-tests config and setting it might potentially collide with other test results. What do you think should be the best approach here?

Reply:
You can place the test in this file `integration-tests/api/__tests__/returns/ff-tax-inclusive-pricing.js` instead - it will have the feature flag enabled :)

Reply:
ahh, totally missed that - added, should be covered now!